components/keyboard-shortcuts/map.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	toggle_editor_mode: '⌘+Shift+Alt+M',
```

Comment:
Is it a good idea to keep "editor"'s specific shortcuts in a generic `components` module? why can't we avoid this map entirely and just pass the shortcut?

Reply:
Yes, I can change it to use the shortcut directly. My thinking was to have a separate list in case it should be used elsewhere (like complete list of shortcuts).

Reply:
Fair enough! let's move the list to the `editor/edit-post` folder then and retrieve passing the mapped value directly to the generic component.

Reply:
Maybe this map could also hold the actual value used to trigger the shortcut `mod+shift+alt+m`. So we'll have a `value` and a `label` per shortcut.